recipes/genomics/Danio_rerio/danRer10/danrer10-gtf-ensembl-v1/meta.yaml

```diff
@@ -0,0 +1,32 @@
+build:
+  binary_relocation: false
+  detect_binary_files_with_prefix: false
+  noarch: generic
+  number: 0
+extra:
+  authors: yliu
+  extra-files: []
+package:
+  name: danrer10-gtf-ensembl-v1
+  version: '1'
+requirements:
+  build:
+  - gsort
+  - htslib
+  - zlib
+  run:
+  - gsort
+  - htslib
+  - zlib
+source:
+  path: .
+about:
+  identifiers:
+    genome-build: danRer10
+    species: Danio_rerio
+  keywords:
+  - gtf
+  summary: GTF gene annotation from Ensembl
```

Comment:
Additional information stating that any regions that do not remap are removed from the file. (This is from your sed command in your recipe)